### PR TITLE
dm: virtio-gpu: update copyright and license

### PR DESCRIPTION
**Pull request by psun3x**
Add license information to virtio_gpu.c
Fix some grammar issues.

Tracked-On: #7241
Signed-off-by: Sun Peng <peng.p.sun@linux.intel.com>
Acked-by: Wang, Yu1 <yu1.wang@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Review by ywan170 — APPROVED** [review]

File: devicemodel/include/vga.h [hunk 1]
```diff
@@ -2,6 +2,9 @@
  * Copyright (C) 2022 Intel Corporation. All rights reserved.
```
Comment (gvancuts) [inline devicemodel/include/vga.h:2]:
```suggestion
 * Copyright (C) 2022 Intel Corporation.
```

File: devicemodel/hw/pci/virtio/virtio_gpu.c [hunk 2]
```diff
@@ -1,6 +1,9 @@
 /*
+ * Copyright (C) OASIS Open 2018. All rights reserved.
  * Copyright (C) 2022 Intel Corporation.
  *
+ * SPDX-License-Identifier: BSD-3-Clause
```
Comment (gvancuts) [inline devicemodel/hw/pci/virtio/virtio_gpu.c:5]: Is this the license that was used in the original file? If not, can we change it?

**Review by gvancuts — CHANGES_REQUESTED** [review]

File: devicemodel/hw/pci/virtio/virtio_gpu.c (hunk 2 above)
Comment (psun3x) [reply devicemodel/hw/pci/virtio/virtio_gpu.c:5]: No, this file just used some definitions about virtio-gpu spec(https://github.com/oasis-tcs/virtio-spec/blob/master/virtio-gpu.tex). We can change this file.

Comment (gvancuts) [reply devicemodel/hw/pci/virtio/virtio_gpu.c:5]: > No, this file just used some definitions about virtio-gpu spec(https://github.com/oasis-tcs/virtio-spec/blob/master/virtio-gpu.tex). We can change this file.

OK, thanks for the confirmation (/cc @yaomichael )

**Review by gvancuts — APPROVED** [review]

**Comment by GGGZZZRRR** [comment]
@psun3x ,please check this error
+ git apply hv.patch
error: patch failed: devicemodel/include/vga.h:2
error: devicemodel/include/vga.h: patch does not apply

**Comment by wenlingz** [comment]
@ywan170 please help approve this PR, thanks

**Review by ywan170 — APPROVED** [review]

**Review by gvancuts — APPROVED** [review]
Thanks!

**Pull request merged**